[State Sync] Update state sync dashboards.

JoshLind opened the pull request:
### Description
This PR updates the state sync dashboards to include a number of newly added metrics, including: (i) compression; (ii) network peers and message overflows; (iii) data stream counts, etc. 

Screenshot below:
<img width="2498" alt="Screen Shot 2022-08-23 at 11 45 26 AM" src="https://user-images.githubusercontent.com/4578587/186202772-0d0e211d-2cf3-4ef1-9cf9-1445f04ac4ed.png">

### Test Plan
Manually inspected the dashboard files and verified them.

<!-- Reviewable:start -->
- - -
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/aptos-labs/aptos-core/3336)
<!-- Reviewable:end -->

sitalkedia commented on dashboards/state-sync-v2.json:
```diff
@@ -3511,8 +4078,8 @@
           "selected": false,
-          "text": "aptos-devnet",
-          "value": "aptos-devnet"
+          "text": "sherryxnet",
```
Is this an intended change?

JoshLind replied:
Annoyingly not. 😭 I made these changes on the `sherryxnet` deployment because devnet monitoring is down and I guess this is what we get when we have to copy/pasta the json. I'll wait for the devnet monitoring pod to come up to re-sync them.

JoshLind commented:
@sitalkedia, I've updated the dashboards with some last minute metrics. I've updated the default to point to devnet via `VictoriaMetrics Global`. This is as good as currently have (the other devnet dashboards are down). Shouldn't be a big issue to change these in the dashboard dynamically anyway.
